lib/utils.js

```diff
@@ -84,18 +84,19 @@ const isoTime = (date, time, zone) => {
 const inputTime = (date, zone) => {
   let tz = zone;
 
-  if (date === '' || date === undefined) {
+  if (date === '' || date === undefined || date === null) {
```

Comment:
Yah because it doesn't work with stuff coming out of the DB for some reason. They're typeof object